src/core_plugins/kibana/public/doc/index.html

```diff
@@ -1,47 +1,75 @@
 <div ng-controller="doc" class="app-container">
+  <div class="kuiLocalNav">
```

Comment:
I hear you, but the LocalNav looks bad in this UI, for the reasons I outlined. I don't think it's a step forward. I know you mentioned in your original comment that you were planning on using it here... I thought it was worth trying it out, since I wasn't sure which content we were going to put inside of it. But now that I see it, I know this is not a good direction. If we use the markup I suggest, I'm 👍 with merging it and using a similar treatment for the Log.